typed_sql/test/sqlite/sqlite_test.dart

```diff
@@ -14,26 +14,38 @@
 
 import 'dart:async';
 
+import 'package:meta/meta.dart' hide literal;
```

Comment:
Unrelated, but this is an annoying conflict.

We should perhaps reconsider some of the names, `literal` is the only function you need to call. Almost everything else is callbacks or extensions.

I suppose you can also do `Expr()` instead of `literal` since I gave `Expr` a factory constructor that calls `literal`.

Reply:
I wouldn't worry about it much: if there is a conflict, people have proper tools to resolve it. We had a lots of back-and-forth with short and maybe conflicting names in `package:postgres`, and short names won.